test/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnerPartnershipTradingNameControllerSpec.scala

```diff
@@ -43,7 +44,7 @@ class BusinessPartnerPartnershipTradingNameControllerSpec extends ControllerSpec
         setupUserAction()
 
         when(mockAppConfig.newBusinessPartnerPagesEnabled).thenReturn(true)
-        val request = FakeRequest()
+        val request = FakeRequest().withCookies(Cookie(name = "businessType", value = ""))
```

Comment:
Add a business type ?

Reply:
Removed the cookie as its not needed here